data.py

```diff
@@ -86,7 +86,7 @@ def __getitem__(self, i) -> Dict[str, torch.Tensor]:
         processor = self.processor
 
         prompt = processor.apply_chat_template(messages, add_generation_prompt=True)
-        inputs = processor(text=prompt, images=[images], return_tensors='pt')
+        inputs = processor(text=prompt, images=images if len(images) > 0 else None, return_tensors='pt')
```

Comment:
`[images] if len(images) > 0 else None` ?

Reply:
nope. do not need double list. Not sure how it worked before (probably Mistral deeps tolerated that)